pkg/activator/handler/context_handler_test.go

```diff
@@ -83,6 +83,51 @@ func TestContextHandlerError(t *testing.T) {
 	}
 }
 
+func BenchmarkContextHandler(b *testing.B) {
+	tests := []struct {
+		label        string
+		revisionName string
+	}{
+		{
+			label:        "context handler success",
+			revisionName: testRevName,
+		},
+		{
+			label:        "context handler failure",
+			revisionName: "fake",
+		},
+	}
+	ctx, cancel, _ := rtesting.SetupFakeContextWithCancel(&testing.T{})
+	defer cancel()
+	revID := types.NamespacedName{Namespace: testNamespace, Name: testRevName}
+	revision := revision(revID.Namespace, revID.Name)
+	revisionInformer(ctx, revision)
+
+	baseHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {})
+
+	handler := NewContextHandler(ctx, baseHandler)
+	req := httptest.NewRequest(http.MethodGet, "http://example.com", nil).WithContext(ctx)
```

Comment:
I don't think we should attach the context here. It's not needed AFAIK.

Reply:
Done.